WasatchNET/EEPROM.cs

```diff
@@ -1851,11 +1857,11 @@ public void setDefault(Spectrometer a)
 
             startupIntegrationTimeMS = 0;
             double temp = 0;
-            startupDetectorTemperatureDegC = (short)temp;
-            if (startupDetectorTemperatureDegC >= 99)
-                startupDetectorTemperatureDegC = 15;
-            else if (startupDetectorTemperatureDegC <= -50)
-                startupDetectorTemperatureDegC = 15;
+            TECSetpoint = (short)temp;
+            if (TECSetpoint >= 99)
+                TECSetpoint = 15;
+            else if (TECSetpoint <= -50)
+                TECSetpoint = 15;

```

Comment:
Hopefully overridden in AndorEEPROM because they definitely go below -50C.

Reply:
I can confirm it is